hack/arktos-up.sh

```diff
@@ -523,7 +523,8 @@ if [[ "${DEFAULT_STORAGE_CLASS}" = "true" ]]; then
 fi
 
 ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f "${KUBE_ROOT}/pkg/controller/artifacts/crd-network.yaml"
-
+# refresh the resource discovery cache after the CRD is created
+${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" api-resources &>/dev/null
```

Comment:
It only works in arktos-up, correct? What about cluster installed by other means such as kubadm or GCE cluster in perf test?

Reply:
The change "${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f "${KUBE_ROOT}/pkg/controller/artifacts/crd-network.yaml" does NOT happen in kubadm or GCE cluster, so this change is not needed.

Reply:
I am confused. Are you saying those will not be supported in automatically set up cluster or those are not currently supported? It looks line 525 and 527 should be used together to support crd network. Isn't it? If so, I would suggest let user to do both kubectl action so that they understand it better. Otherwise, Arktos is giving wrong impression of how those are used.

Reply:
synced offline. This action only happens in dev cluster only so far. Not ready for generic clusters yet.